risu.go

```diff
@@ -2,19 +2,27 @@ package main
 
 import (
 	"encoding/json"
+	"fmt"
 	"log"
 	"net/http"
+	"os"
 	"time"
 
 	"code.google.com/p/go-uuid/uuid"
 	"github.com/codegangsta/negroni"
 	"github.com/julienschmidt/httprouter"
+	"github.com/libgit2/git2go"
 	"github.com/unrolled/render"
 
 	"github.com/wantedly/risu/registry"
 	"github.com/wantedly/risu/schema"
 )
 
+const (
+	GitHubHost           = "github.com/"
```

Comment:
GitHubHost = "github.com/" seems so obvious and I think this is not what you want to describe.
Why don't you use `GitRepositoryHost`?